sharing_portal/urls.py

```diff
@@ -10,6 +10,9 @@
     path("project/<str:charge_code>", views.index_project, name="index_project"),
     path("<int:pk>", views.artifact, name="detail"),
     path("<int:pk>/launch", views.launch, name="launch"),
+    path("<int:pk>/request", views.request_day_pass, name="request_day_pass"),
+    path("list_requests", views.list_day_pass_requests, name="list_day_pass_requests"),
```

Comment:
Nit: in general our paths follow REST conventions, so this would just be `path("requests", ...)`